.github/workflows/build-and-run-model.yaml

```diff
@@ -14,14 +14,29 @@ on:
     types: [opened, reopened, synchronize, closed]
   workflow_dispatch:
     inputs:
-      enable_upload:
+      run_note:
+        type: string
+        description: Note to include with run
+        required: true
+      upload_enable:
         type: boolean
         description: Upload results to S3
+        default: true
+        required: true
+      cv_enable:
```

Comment:
**[Thought, non-blocking]** One possible alternative to this approach: If we think the list of override env vars might continue to grow such that these params become unwieldy, we could consider a single `env_overrides` attribute that accepts a newline- (or comma-) delimited list of additional env vars to set. But I think if we don't anticipate adding any more env var overrides, then splitting each one out into its own param makes sense to me!

Reply:
I think (hope) this is the extent of the override parameters. I don't foresee any other toggleable features soon.